core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1376,6 +1376,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_JOURNAL_TOLERATE_CORRUPTION =
+      new Builder(Name.MASTER_JOURNAL_TOLERATE_CORRUPTION)
+          .setDefaultValue(false)
```

Comment:
can we mark this as hidden so that it doesn't appear in the docs?

Reply:
Marked as hidden, thanks